test/lib/common.js

```diff
@@ -0,0 +1,73 @@
+/*
+* © Copyright IBM Corp. 2018
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+ * Contains helper code that is used by the tests.
+ */
+'use strict';
+
+// Paths to the generated files, if you move files around change paths here.
+/*eslint-disable */
+exports.commonFile = {
+  server_go: 'server.go',
+  server_test_go: 'routers/health_test.go',
+  gitignore: '.gitignore',
+  README_md: 'README.md',
+  Gopkg_lock: 'Gopkg.lock',
+  Gopkg_toml: 'Gopkg.toml',
+  health_go: 'routers/health.go',
+  run_dev: 'run-dev'
+}
+
+exports.webAppFile = {
+  index_go: 'routers/index.go',
```

Comment:
if this is a list of files expected for a web app, include index.html, 500.html and 404.html and then remove the next section, lines 39-43

Reply:
Go-server is the one that composes with the web generator, so we test for the html files in that generator.